addon/components/md-checkbox.js

```diff
@@ -13,22 +14,18 @@ var MdCheckbox = Ember.Component.extend(LayoutRules, RippleMixin, {
 
     attributeBindings: ['isDisabled:disabled', 'aria-label', 'mdNoInk', 'tabindex'],
     tabindex: 0,
+    checked: false,
+
+    didInsertElement() {
+        this._super(...arguments);
+        setupRipples(this, this.$('.md-container'));
```

Comment:
This doesn't seem like the correct behavior. We want to call attachCheckboxBehavior, not attachButtonBehavior, which isn't exposed in the setupRipples utility.